crates/evm/evm/src/inspectors/debugger.rs

```diff
@@ -51,11 +50,14 @@ impl<DB: DatabaseExt> Inspector<DB> for Debugger {
         let op = interp.current_opcode();
 
         // Get opcode information
-        let opcode_infos = spec_opcode_gas(ecx.spec_id());
-        let opcode_info = &opcode_infos[op as usize];
-
+        let _opcode_info = OPCODE_INFO_JUMPTABLE[op as usize]
+            .ok_or_else(|| eyre::eyre!("Invalid opcode: {}, Not found in jump table", op));
```

Comment:
Let's remove these jumptable calls if we're not going to use the result. In EOF, more opcodes will have immediate values which we need to skip, but the entire iterator will look different anyway, so the manual check for push opcode is fine.